### PR TITLE
Add Windows process aggregation

### PR DESCRIPTION
**Pull request by nicholasberlin**
## Change Summary

Changes are for aggregation of start and exit process events for Windows.

## Release Target

8.16

### For mapping changes:

- [x] I ran `make` after making the schema changes, and committed all changes

Chained off of: https://github.com/elastic/endpoint-package/pull/544

### CODE REVIEW

File: custom_documentation/doc/endpoint/process/windows/windows_process_create_and_exit.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ This event is generated when a process is created.
 | process.command_line |
 | process.entity_id |
 | process.executable |
+| process.exit_code |
```
Comment (gabriellandau) [inline custom_documentation/doc/endpoint/process/windows/windows_process_create_and_exit.md:90]: In an aggregated event, do we lose the timestamp of process termination?

**Review by pzl — COMMENTED** [review]
if this change is only affecting the documentation, then you are good to merge.  defend workflows doesn't need to approve anything in custom docs

File: custom_documentation/doc/endpoint/process/windows/windows_process_create_and_exit.md (hunk 1 above)
Comment (nicholasberlin) [reply custom_documentation/doc/endpoint/process/windows/windows_process_create_and_exit.md:90]: Events are aggregated in the order they are received. Aggregation entails overwriting all fields except for those marked for aggregation, which is currently only [`event.type`](https://github.com/elastic/endpoint-dev/blob/086511efc813328e9c5062977e788bc6dcfdb2bf/Python/endpoint/generators/events/v0/events.yml#L321) and [`event.action`](https://github.com/elastic/endpoint-dev/blob/086511efc813328e9c5062977e788bc6dcfdb2bf/Python/endpoint/generators/events/v0/events.yml#L360). So, `@timestamp` of the aggregated event will be the `@timestamp` from the process termination event, and the aggregated `event.action` will be `["start", "end"]`.

So, to answer your question, no. But, we do lose the `@timestamp` from the process creation. `process.start` will be overwritten as well, but I assume that `process.start` of a termination event is equal to the corresponding creation event.

Comment (gabriellandau) [reply custom_documentation/doc/endpoint/process/windows/windows_process_create_and_exit.md:90]: Okay, so it sounds like both timestamps will be present in the aggregated event.  Thanks.

**Review by gabriellandau — APPROVED** [review]

**Pull request merged**